### PR TITLE
Adding Word controller

### PR DESCRIPTION
**Pull request by Rownaknira**

### CODE REVIEW

File: app/controllers/admin/words_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  end
+
+  def destroy
+    Word.find(params[:id]).destroy
+    redirect_to words_url
+  end
+
+  private
+  def word_params
+    params.require(:word).permit(:category_id, :content)
```
Comment (chuanhtuan) [inline app/controllers/admin/words_controller.rb]: please nested answer in creating word screen
there are 4 answer for each word

File: app/controllers/admin/words_controller.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  def new
+    @word = Word.new
+  end
+
+  def create
+    @word = Word.new word_params
+    if @word.save
+      redirect_to words_path
+    else
+      render 'new'
```
Comment (chikim) [inline app/controllers/admin/words_controller.rb]: use symbol instead of string

File: app/controllers/admin/words_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  def edit
+    @word = Word.find params[:id]
+  end
+
+  def update
+    @word = Word.find params[:id]
+    if @word.update_attributes word_params
+      redirect_to words_path
+    else
+      render 'edit'
```
Comment (chikim) [inline app/controllers/admin/words_controller.rb]: use symbol instead of string

File: app/controllers/admin/words_controller.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      render 'new'
+    end 
+  end
+
+  def edit
+    @word = Word.find params[:id]
+  end
+
+  def update
+    @word = Word.find params[:id]
```
Comment (chikim) [inline app/controllers/admin/words_controller.rb]: pls create a before_action only: [:edit, :update, :destroy] to DRY this

File: app/controllers/admin/words_controller.rb [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    @word = Word.new word_params
+    if @word.save
+      redirect_to words_path
+    else
+      render 'new'
+    end 
+  end
+
+  def edit
+    @word = Word.find params[:id]
```
Comment (chikim) [inline app/controllers/admin/words_controller.rb]: pls create a before_action only: [:edit, :update, :destroy] to DRY this

File: app/controllers/admin/words_controller.rb [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    if @word.update_attributes word_params
+      redirect_to words_path
+    else
+      render 'edit'
+    end    
+  end
+
+  def destroy
+    Word.find(params[:id]).destroy
```
Comment (chikim) [inline app/controllers/admin/words_controller.rb]: pls create a before_action only: [:edit, :update, :destroy] to DRY this

File: app/views/admin/words/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+      <tr>
+        <td><%= word.category.name%></td>
+        <td><%= word.content%></td>
+        <td><%= link_to t('form.edit'),edit_admin_word_path(word)%></td>
+        <td><%= link_to t('form.delete'),admin_word_path(word),method: :delete,data: {confirm: t('delete.confirmation')}%></td>
+      </tr>
+    <% end %>
+  </table>
+  <br>
+    <%= link_to t('form.create'),new_admin_word_path,class: "btn btn-primary"%>
```
Comment (chikim) [inline app/views/admin/words/index.html.erb]: indent

File: app/assets/stylesheets/words.scss [hunk 8]
```diff
@@ -0,0 +1 @@
+
```
Comment (chikim) [inline app/assets/stylesheets/words.scss]: what is purpose of this file?

File: app/controllers/admin/categories_controller.rb [hunk 9]
```diff
@@ -1,32 +1,43 @@
...
-      redirect_to categories_path
+      redirect_to admin_categories_path
     else
-      render 'new'
+      render :new
     end    
   end
+
   def edit
     @category = Category.find params[:id]
```
Comment (chuanhtuan) [inline app/controllers/admin/categories_controller.rb]: use a load_object in before_action

File: app/controllers/admin/categories_controller.rb [hunk 10]
```diff
@@ -1,32 +1,43 @@
...
-    redirect_to categories_url
+    redirect_to admin_categories_url
   end
+
   private
-    def category_params
-      params.require(:category).permit(:name)
-    end
+  def category_params
+    params.require(:category).permit(:name)
```
Comment (chuanhtuan) [inline app/controllers/admin/categories_controller.rb]: ()

**Pull request merged**